### ISSUE TITLE
bosh-lite Instructions Don't Work

### ISSUE THREAD

**Issue by DanielJonesEB** [cloudfoundry-community/postgresql-docker-boshrelease#13]
```
$ templates/make_manifest warden container embedded
Acting as user 'admin' on 'Bosh Lite Director'
Bosh cpi 9bca5701-66c0-43ee-a4d0-64813b4b5ece bosh-warden-boshlite-ubuntu-trusty-go_agent
12 error(s) detected:
 - $.compilation.cloud_properties: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.director_uuid: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.jobs: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.meta.job_templates: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.meta.name: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.meta.stemcell: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.meta.subnets: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.networks: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.networks: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.properties: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.releases: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)
 - $.resource_pools.0.cloud_properties: inappropriate use of (( merge )) operator outside of a list (this is spruce, after all)


$ spruce --version
spruce - Version 1.0.1 (release)
$ git rev-parse HEAD
85f48710145fbec4f5504c595c20b545f0cfe2d1
```

**Comment by drnic** [comment]
Pooey to half migrated spiff -> spruce templates :/

**Comment by DanielJonesEB** [comment]
Fixed by #14
